doc/man1/s_client.pod

```diff
@@ -132,6 +135,18 @@ When used with the B<-connect> flag, the program uses the host and port
 specified with this flag and issues an HTTP CONNECT command to connect
 to the desired server.
 
+=item B<-unix val>
+
+Connect over Unix domain sockets.
```

Comment:
Perhaps "path" instead of val, and "Connect over the specified Unix-domain socket"